docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -51,50 +52,84 @@ Create a new fee juice portal manager and bridge fee juice publicly to Aztec:
 
 #include_code bridge_fee_juice yarn-project/end-to-end/src/spartan/setup_test_wallets.ts javascript
 
-Bridging can also be done privately with the corresponding function:
+For the mechanisms to complete bridging between L1 and Aztec, we have to wait for 2 Aztec blocks to progress. This can be triggered manually by sending 2 transactions in the sandbox, or by waiting for 2 blocks on a public network. After this, an account should have its fee juice ready to use in transactions.
 
-#include_code bridge_tokens_private yarn-project/aztec.js/src/ethereum/portal_manager.ts javascript
+Alternatively, the resulting claim object can be used to create a payment method to claim and pay for a transaction in one, where the transaction is the contract's deployment.
 
-For the mechanisms to complete bridging between L1 and Aztec, any two other transactions on the sandbox are made. After this, an already deployed account should have its fee juice ready to use in transactions.
+## Examples
 
-Alternatively, the resulting claim object can be used to create a payment method to claim and pay for a transaction in one, where the transaction is the contract's deployment.
+### Pay with FeeJuice
 
+To send a transaction from an already deployed account already holding fee juice. A user's fee juice balance is public, so paying for a transaction with fee juice will reveal information about which account is paying for the transaction and how much they paid.
 
-## Fee Options
+Note: this example is a public token transfer call, but can equally be a private function call
 
-Functions pertaining to sending a transaction, such as `deploy` and `send`, each include a `fee` variable defined with the following (optional) parameters:
+```ts
```

Comment:
hmm I guess this doesn't exist anywhere in the codebase?

Reply:
not an import statement that is just this.